### ISSUE TITLE
EPUB Exports: Language attribute is missing (toc.xhtml)

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4444]
The `<html>` element does not have a `lang` attribute.

[Daisy Accessible Publishing Knowledge Base](https://kb.daisy.org/publishing/docs/html/lang.html)

HTML element in toc.xhtml currently:
```
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops">
```

Could be changed to:
```
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:epub="http://www.idpf.org/2007/ops"
      lang="en-CA"
      xml:lang="en-CA">
```

**Comment by SteelWagstaff** [comment]
We attempt to set the lang value in the toc here: https://github.com/pressbooks/pressbooks/blob/4ef0ff44e90e41a1968948cfd121cbd5c515c2eb/inc/modules/export/epub/class-epub.php#L1892. Is this not appearing in the finished HTML exports? If not, can we confirm that it has been set correctly in the book in question?
